app/team_test.go

```diff
@@ -729,3 +729,37 @@ func TestGetTeamMembers(t *testing.T) {
 		assert.Equal(t, userIDs[i], member.UserId)
 	}
 }
+
+func TestGetTeamStats(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+
+	var userIDs sort.StringSlice
+	userIDs = append(userIDs, th.BasicUser.Id)
+	userIDs = append(userIDs, th.BasicUser2.Id)
+
+	for i := 0; i < 2; i++ {
+		user := model.User{
+			Email:    strings.ToLower(model.NewId()) + "success+test@example.com",
+			Username: fmt.Sprintf("user%v", i),
+			Password: "passwd1",
+		}
+		ruser, err := th.App.CreateUser(&user)
+		require.Nil(t, err)
+		require.NotNil(t, ruser)
+		defer th.App.PermanentDeleteUser(&user)
+
+		_, err = th.App.AddUserToTeam(th.BasicTeam.Id, ruser.Id, "")
+		require.Nil(t, err)
+
+		// Store the user ids for comparison later
+		userIDs = append(userIDs, ruser.Id)
+	}
+	// Sort them because the result of GetTeamMembers() is also sorted
```

Comment:
This looks like a copy paste from the team above: but we're not invoking `GetTeamMembers`, right? Can we dispense with `userIDs` altogether and just assert the total member count as such?

Ideally, we'd also create users on another team and verify that we get appropriate results by team.

That being said, this is just a thin layer over top of the original `GetTeamStats`, which should have been unit tested anyway, so I'm 0/5 as to whether we need a unit tests at this layer at all. (Much more interesting would be a plugin that uses this API to prove that the whole pipeline works.)